src/app/machines/views/MachineList/MachineListControls/MachineListControls.tsx

```diff
@@ -27,42 +39,138 @@ const MachineListControls = ({
   setHiddenGroups,
   hiddenColumns,
   setHiddenColumns,
+  setSidePanelContent,
 }: MachineListControlsProps): JSX.Element => {
   const [searchText, setSearchText] = useState(filter);
+  const hasSelection = useHasSelection();
+  const sendAnalytics = useSendAnalytics();
+  const dispatch = useDispatch();
+  const [tagsSeen, setTagsSeen] = useStorageState(
+    localStorage,
+    "machineViewTagsSeen",
+    false
+  );
 
   useEffect(() => {
     // If the filters change then update the search input text.
     setSearchText(filter);
   }, [filter]);
 
+  const getTitle = useCallback(
+    (action: NodeActions) => {
+      if (action === NodeActions.TAG) {
+        const title = getNodeActionTitle(action);
+        if (!tagsSeen) {
+          return (
+            <>
+              {title} <i className="p-text--small">(NEW)</i>
+            </>
+          );
+        }
+        return title;
+      }
+      return null;
+    },
+    [tagsSeen]
+  );
+
   return (
     <Row className="machine-list-controls">
-      <Col size={3}>
-        <MachinesFilterAccordion
-          searchText={searchText}
-          setSearchText={(searchText) => {
-            setFilter(searchText);
-          }}
-        />
-      </Col>
-      <Col size={4}>
-        <DebounceSearchBox
-          onDebounced={(debouncedText) => setFilter(debouncedText)}
-          searchText={searchText}
-          setSearchText={setSearchText}
-        />
-      </Col>
-      <Col size={3}>
-        <div className="u-flex--align-baseline">
-          <div className="u-flex--grow">
-            <GroupSelect
-              grouping={grouping}
-              setGrouping={setGrouping}
-              setHiddenGroups={setHiddenGroups}
+      {!hasSelection ? (
```

Comment:
This means that filter groups are fetched again every time you clear the selection.

![Kapture 2023-03-29 at 10 56 26](https://user-images.githubusercontent.com/7452681/228495369-f98304d1-96a3-4d70-b9f5-14360c79457d.gif)

This can be mitigated by adding a check for whether filter groups have been loaded before dispatching the `machineActions.filterGroups` action in MachinesFilterAccordion component.

I added a separate task for this: https://warthogs.atlassian.net/browse/MAASENG-1522